Lazydocs: API Reference doc refresh (#1424)
wandb/docs: content/en/guides/models/sweeps/initialize-sweeps.md

{{% alert color="secondary" %}}
1. Before you initialize a sweep, make sure you have a sweep configuration defined either in a YAML file or a nested Python dictionary object in your script. For more information see, [Define sweep configuration]({{< relref "/guides/models/sweeps/define-sweep-configuration.md" >}}).

> 2. Both the W&B Sweep and the W&B Run must be in the same project. Therefore, the name you provide when you initialize W&B ([`wandb.init`]({{< relref "/ref/python/sdk/functions/init" >}})) must match the name of the project you provide when you initialize a W&B Sweep ([`wandb.sweep`]({{< relref "/ref/python/sdk/functions/sweep" >}})).

Reviewer: ```suggestion
2. The W&B Sweep and the W&B Run must be in the same project. Pass the same project name when you initialize W&B ([`wandb.init`]({{< relref "/ref/python/sdk/functions/init" >}})) and when you initialize a W&B Sweep ([`wandb.sweep`]({{< relref "/ref/python/sdk/functions/sweep" >}})).
```
